vbeam/data_importers/pyuff_importer.py

```diff
@@ -152,25 +155,26 @@ def import_pyuff(
 
     # Check if we are dealing with a STAI dataset: is each virtual source placed at
     # exactly at an element position?
-    if receivers.position.shape == wave_data.source.shape and (
-        numpy.allclose(receivers.position, wave_data.source)
+    if receiver.shape == wave_data.source.shape and (
+        numpy.allclose(probe.receiver_position, wave_data.source)
     ):
         # We are dealing with a STAI dataset! Senders are each element in the array
-        sender = receivers.copy()
+        probe.sender_position = probe.receiver_position.copy()
```

Comment:
Is `receiver_position` set on the probe object here?

Reply:
This was an old implementation. I have updated this now.